src/webview/MessageList.js

```diff
@@ -102,6 +103,7 @@ export type Props = $ReadOnly<{|
   narrow: Narrow,
   showMessagePlaceholders: boolean,
   startEditMessage: (editMessage: EditMessage) => void,
+  startReplyWithMention: (replyWithMention: ReplyWithMention) => void,
```

Comment:
Why does this prop get passed into `MessageList`, when it doesn't seem to use it?

Reply:
That seems odd to me! I don't recall why!

Reply:
That's make me wonder, why `startEditMessage` passed to MessageList?